onnxruntime/core/providers/cuda/cuda_execution_provider_info.h

```diff
@@ -11,12 +11,22 @@
 #include "core/session/onnxruntime_c_api.h"
 
 namespace onnxruntime {
+namespace cuda {
+namespace provider_option_names {
+constexpr const char* kDeviceId = "device_id";
```

Comment:
Can these consts be scoped to the cc file only?

Reply:
ok. i thought they could be useful elsewhere, but they aren't used now.

---
In reply to: [546978143](https://github.com/microsoft/onnxruntime/pull/6171#discussion_r546978143) [](ancestors = 546978143)